frontend/src/scenes/plugins/tabs/apps/AppManagementView.tsx

```diff
@@ -78,7 +80,7 @@ export function AppManagementView({
                                 }
                                 data-attr="plugin-uninstall"
                             >
-                                Uninstall
+                                Uninstall me
```

Comment:
why the "me", in any case this is the older UI that will be replaced by pipeline folder

Reply:
Whoops! That was just me testing to make sure I was looking at the correct button. Removed in the latest commit